data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+      _([[The heavy weight plus the strong current was too much for the ship and it was pulled below the sea level by Lutas forces and we lost most men and all wares it carried.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Jundlina - Diary
+      _([[The fear was great in us all. We expected the other ships to be drowned as well, but they persisted. We escaped the island and began our smooth sailing to other coasts. ]]) ..
+      _([[Many weeks have passed and we did only see small islands not suitable for habitation and without any sign of being blessed by Satul, the fire god. ]]) ..
+      _([[Last night however, we made out a new island. We saw smoke hovering over it. The black comes from the fiery mountain in the north of the island, the grey however comes from settlers.]])),
+      h=500,
+   },
+   {
+      title = _"Planning for the future",
+      body = jundlina(_"Jundlina continues",
+      -- TRANSLATORS: Jundlina - Diary
+      _([[At least the east side of the island is inhabited, so much is clear. We do not know if the people there are friends or foes. We do not even know what tribe they might be from. ]]) ..
+      _([[The only thing that is sure that they can’t be Atlanteans for no ship of us has sailed as far as us in the last hundreds years. I can only hope they are friendly. ]]) ..
+      _([[We have not much water left, our food is running low, we lost all our tools and most of our building materials with the third ship. We can’t afford sail any further an need to try to settle ourselves on this island.]]))
+      .. new_objectives(obj_find_port),
+      h=500,
+   },
+} -- end of initial messages.
+
+initial_messages1= {
+   {
+      title = _"Jundlina gives orders",
+      body = jundlina(_"Jundlina decides",
+      -- TRANSLATORS: Jundlina - Diary
+      _([[As we founded a port here, we can now plan our settlement. We still have plenty of iron ore, so a metal workshop and a smelter will provide us with good tools in no time. Also, we need to solve our building material problem. ]]) ..
+      _([[The island is full of trees, we might not even need foresters. But we need stone, that is for sure!]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Jundlina - Diary
+      _([[Let’s also explore this island. But we need to be careful. We should build at least one tower so that we can see potential enemies before they see us. ]]) ..
+      _([[But we need to be careful with the stones and other precious material though. When we no longer need a building, we will dismantle it instead of burning it down. ]]) ..
+      _([[This will take more time, but we can reuse some of its materials. Also, I want a scout out and exploring at all times.]]))
+      .. new_objectives(obj_tools)
+      .. new_objectives(obj_basic_infrastructure)
+      .. new_objectives(obj_explore),
+      h=500,
+   },
+} -- end of initial messages.
+
+spidercloth_1 = {
+   {
+      title = _"Opol arrives at Jundlina",
```

Comment:
```suggestion
      title = _"Opol visits Jundlina",
```
You can only arrive at a location, not at a person